test/__snapshots__/main.test.ts.snap

```diff
@@ -1816,7 +1816,7 @@ export default function getRoutes(config: RouteConfig) {
 
         },
         data: {
-          query
+          query: JSON.stringify(query)
```

Comment:
sure about this?

Reply:
i guess `query` is a query param, dunno in which project we called it `query` but it could be named anything

Reply:
mmm I'm confused

Reply:
I'm talking about the `JSON.stringify`, see my comment below.